l2geth/core/genesis.go

```diff
@@ -550,10 +516,8 @@ func DeveloperGenesisBlock(period uint64, faucet, l1XDomainMessengerAddress comm
 		// be added into the initial genesis state at runtime
 		L1CrossDomainMessengerAddress: l1XDomainMessengerAddress,
 		L1FeeWalletAddress:            l1FeeWalletAddress,
-		AddressManagerOwnerAddress:    addrManagerOwnerAddress,
 		GasPriceOracleOwnerAddress:    gpoOwnerAddress,
 		L1StandardBridgeAddress:       l1StandardBridgeAddress,
-		ChainID:                       config.ChainID,
```

Comment:
Since this is removed, how are we configuring the chain ID now?

Reply:
This only passed it though to the state, the chainid is still configured, will add a comment to make this obvious